Library/ViewModels/SearchViewModel.swift

```diff
@@ -185,10 +186,27 @@ public final class SearchViewModel: SearchViewModelType, SearchViewModelInputs,
 
     self.scrollToProjectRow = self.transitionedToProjectRowAndTotalProperty.signal.skipNil().map(first)
 
-    // KSRAnalytics
+    // Tracking
 
-    viewWillAppearNotAnimated
-      .observeValues { AppEnvironment.current.ksrAnalytics.trackProjectSearchView() }
+    let projectsProperty = Property(
```

Comment:
The general convention of this project is that we do not define properties unless they are explicitly defined as an output. Let's aim for a cleaner solution.